wiki/Difficulties/osu!/en.md

```diff
@@ -0,0 +1,10 @@
+# osu!
+
+These are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).
+The articles below explains how to setup your beatmap for a specific difficulty and what you should be aware of when making them.
```

Comment:
"setup" is a noun, "set up" is a verb. It should be "how to set up your beatmap".